lib/NetifDriver.swift

```diff
@@ -192,4 +193,20 @@ class NetifDriver : NSObject, ZitiUnretained {
         }
         return tunnelProvider.deleteRoute(destStr)
     }
+    
+    static let exclude_rt_cb:exclude_route_fn = { handle, loop, dest in
+        guard let mySelf = zitiUnretained(NetifDriver.self, UnsafeMutableRawPointer(handle)) else {
+            log.wtf("invalid handle", function: "exclude_rt_cb()")
+            return -1
+        }
+        guard let tunnelProvider = mySelf.tunnelProvider else {
+            log.wtf("invalid tunnelProvider", function: "exclude_rt_cb()")
+            return -1
+        }
+        guard let destStr = dest != nil ? String(cString: dest!) : nil else {
+            log.error("Invalid dest paramater", function: "exclude_rt_cb()")
```

Comment:
so the other two are wtf but this isn't? seems inconsistent but i assume there's 'rationale'?

Reply:
I use WTF on stuff that I set up internally in the framework that indicates programming error if incorrect (e.g., pointer conversions for moving between Swift and C), and error everywhere else.